federation/ipatuura/README.md

```diff
@@ -187,12 +193,24 @@ The `integrationdomain.json` file looks like this:
 
 4- `"keycloak_hostname"` must resolve to the host running keycloak (see step above to ensure it was added to `/etc/hosts`)
 
+## Setup outgoing certificates
```

Comment:
Without these below steps when adding the 'ipatuura' plugin in Keycloak it would fail with

~~~
 ERROR [org.keycloak.ipatuura_user_spi.Ipatuura] (executor-thread-4) Error: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
~~~